### DISCUSSION TITLE
Error: Shared module is not available for eager consumption

### DISCUSSION THREAD

**Discussion by xcqbb** [webpack/webpack#15805] [Q&A]
How should I handle shared modules containing vue?

**Comment by vankop** [comment]
same as any other js..

**Reply by xcqbb** [reply]
I added 'eager: true' to vue, is it wrong?

**Reply by xcqbb** [reply]
const dependencies = require("./package.json").dependencies;
const shared = {
  vue: {
    eager: true,
    import: 'vue',
    shareKey: 'shared-vue',
    shareScope: 'default',
    requiredVersion: dependencies.vue,
    singleton: true,
  },
}
